src/components/aeIdentityAvatar/aeIdentityAvatar.vue

```diff
@@ -2,6 +2,8 @@
   <div :style='style' class="avatar ae-identity-avatar"></div>
 </template>
 
-<script src='./aeIdentityAvatar.js'/>
+<script src='./aeIdentityAvatar.js'>
+  export { default } from './aeIdentityAvatar'
```

Comment:
Is it necessary to add this?

Reply:
Yes, the unit tests don't import it right if you don't have it (sadly).